csrc/fusion_segmenter.cpp

```diff
@@ -4080,7 +4080,7 @@ SegmentCandidateFinder::SegmentCandidateFinder(
     runtime_info_.emplace(segmented_fusion_->completeFusion(), inputs);
   }
 
-  privatizeUpcast();
+  privatizeUpCastAndSqueeze();
```

Comment:
It's likely we will add more ops, so let's rename the function this time to something like `privatizeOps`.